cruise-control-metrics-reporter/src/main/java/com/linkedin/kafka/cruisecontrol/metricsreporter/CruiseControlMetricsReporterConfig.java

```diff
@@ -67,7 +64,7 @@
   public static final Short DEFAULT_CRUISE_CONTROL_MIN_INSYNC_REPLICAS = -1;
   public static final long DEFAULT_CRUISE_CONTROL_METRICS_REPORTER_INTERVAL_MS = 60000;
   public static final String PRODUCER_ID = "CruiseControlMetricsReporter";
-  public static final int DEFAULT_CRUISE_CONTROL_METRICS_REPORTER_LINGER_MS = 30 * 1000;
+  public static final int DEFAULT_CRUISE_CONTROL_METRICS_REPORTER_LINGER_MS = 5 * 1000;
```

Comment:
I highly recommend using `TimeUnit` to set default values for all time-related parameters. For example `DEFAULT_CRUISE_CONTROL_METRICS_REPORTER_LINGER_MS  = TimeUnit.SECONDS.toMillis(5);`

Reply:
I will close this issue and I will create another PR to fix this issue.